run.go

```diff
@@ -92,6 +92,11 @@ func run(context *cli.Context) error {
 		if err != nil {
 			return fmt.Errorf("Process state %s: %s", ps.String(), err)
 		}
+
+		// delete container's resources
+		if err := delete(pod.GetAllContainers()[0].ID(), true); err != nil {
```

Comment:
If we guarantee that GetAllContainers() will indeed return a non empty array with a non nil element after create, not checking sounds fair enough. I wonder if we do guarantee that (not looked).